client/src/legacy/TinyMCE_sslink.js

```diff
@@ -33,33 +43,18 @@ const plugin = {
       }
     }
 
+    // Button in main toolbar
     editor.ui.registry.addMenuButton('sslink', {
       icon: 'link',
       tooltip: titleWithShortcut,
-      fetch: (callback) => callback(
-        // Fetch the actions whenever the fetch function is called.
-        TinyMCEActionRegistrar.getSortedActions('sslink', editor.getParam('editorIdentifier'), true)
-          .map(action => Object.assign(
-            {},
-            action,
-            { onAction: () => action.onAction(editor) }
-          ))
-      ),
+      fetch: (callback) => callback(getActions()),
     });
 
-    editor.on('preinit', () => {
```

Comment:
preinit isn't needed for CMS 5 since TinyMCE 6 provides a lazyloading callback for fetching menu items